src/packages/ratmama/ratmama_parser.py

```diff
@@ -167,39 +172,48 @@
         sep = '-'
 
     if not sep:
-        ctx.bot.board.append(Rescue(irc_nickname=ctx.user.nickname, client=ctx.user.nickname))
+        ctx.bot.board.append(Rescue(irc_nickname=ctx.user.nickname.split("[")[0],
+                                    client=ctx.user.nickname))
         index = ctx.bot.board.find_by_name(ctx.user.nickname)
         await ctx.reply(f"Case #{index} created for {ctx.user.nickname}, please set details")
         return
     system: str = None
-    code_red: bool = False
+    cr: bool = False
     platform: Platforms = None
     for part in message.split(sep):
         part = part.strip()
         if part.casefold() in ("pc",):
             platform = Platforms["PC"]
+            message = re.sub(part, "", message, flags=re.IGNORECASE)
 
         elif part.casefold() in ("ps", "ps4", "playstation", "playstation4", "playstation 4"):
             platform = Platforms["PS"]
+            message = re.sub(part, "", message, flags=re.IGNORECASE)
 
         elif part.casefold() in ("xb", "xb1", "xbox", "xboxone", "xbox one"):
             platform = Platforms["XB"]
+            message = re.sub(part, "", message, flags=re.IGNORECASE)
 
         elif "o2" in part.casefold():
-            code_red = "o2 ok" not in part.casefold()
+            cr = "o2 ok" not in part.casefold()
+            message = re.sub(part, "", message, flags=re.IGNORECASE)
 
-        else:
-            system = part
+    system = message.strip(f"{sep} ")
 
     rescue = Rescue(
-        client=ctx.user.nickname,
+        client=ctx.user.nickname.split("[")[0],
         system=system,
         irc_nickname=ctx.user.nickname,
-        code_red=code_red,
+        code_red=cr,
         platform=platform
     )
+    landmark = await Galaxy.find_nearest_landmark(
+        (await Galaxy.find_system_by_name(system))
```

Comment:
[Error] Same concern as above. If the system does not resolve into a `StarSystem` object, `find_nearest_landmark` will break.